publish.yml

```diff
@@ -0,0 +1,47 @@
+name: publish-evaluate-artifact-policies-$(SourceBranchName)_$(Date:MMMd)$(Rev:.r)
+
+jobs:
+- job: Build
+  timeoutInMinutes: 20
+  cancelTimeoutInMinutes: 2
+  pool:
+    name: Azure Pipelines
+    demands: msbuild
+
+  steps:
+    - task: NuGetToolInstaller@1
+      displayName: 'Install NuGet 4.3.0'
+      inputs:
+        versionSpec: 4.3.0
+        checkLatest: true
+
+    - task: NuGetCommand@2
+      displayName: 'Run NuGet restore on all csproj'
+      inputs:
+        restoreSolution: '**/*.csproj'
+    
+    - task: MSBuild@1
+      displayName: 'Build dirs.proj'
+      inputs:
+        solution: dirs.proj
+        configuration: Release
+        clean: true
+
+    - task: VSTest@2
+      displayName: 'Run unit tests'
+      inputs:
+        testAssemblyVer2: 'Tests/bin/Release/**/*.dll'
+    
+    - task: ArchiveFiles@2
+      displayName: 'Archive azure function'
+      inputs:
+        rootFolderOrFile: AzureFunction/bin/Release/netstandard2.0/
+        includeRootFolder: false
+        archiveFile: '$(Build.ArtifactStagingDirectory)/$(Build.BuildNumber).zip'
+
```

Comment:
Can you add one more step to upload zip has build artifact. Mostly for testing we can download from there and validate.